classes/local/form_handler.js

```diff
@@ -0,0 +1,60 @@
+require(['jquery'], function($) {
```

Comment:
For what purpose is this separate javascript needed?
Could you add some comments, please?
Also Javascript should not be placed here! Have a look at AMD modules! https://docs.moodle.org/dev/Javascript_Modules

Reply:
Thanks, for moving this code to an AMD module. But I still do not know, why you added this javascript. The autocomplete and the filepicker modules should work through javascript from the moodle core. Usually, there is no need to add custom Javascript.

Reply:
There are 2 reasons to have that javascript:
1. In case user upload a large video file via drag & drop into the filepicker, it will take sometimes to upload the file, meanwhile the Save Changes button is enable and can submit the form, therefore, our opencast team came up with the idea to disable/enable the button in this situation.
2. The autocomplete form won't take the css classes from the form-control when create it with no suggestion feature, therefore it looks ugly and makes inconsistency in UI.

Reply:
ok. Thanks for clarifying. I just build the amd module and also added the minified version of it.